### PR TITLE
Add note about pip install editable omero-py

### PR DESCRIPTION
**Pull request by manics**
If you `pip install e .` in omero-py and/or omero-web some omero-web tests may fail due to `omero.plugins.web` not being found

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -51,7 +51,8 @@ For a development installation we recommend creating a virtualenv with the follo
     pip install -e .
 
 This will install OMERO.web into your virtualenv as an editable package, so any edits to source files should be reflected in your installation.
-Note that if you add or remove files you must rerun the last step.
```
Comment (mtbc) [inline README.rst]: Obvious enough to Python developers I guess?

Comment (joshmoore) [reply README.rst]: No, it's that that only holds for the omero-py repo.

**Comment by joshmoore** [comment]
If you `pip install` web but `pip install -e` omero-py, then the error is:

```
_______________________________________________ ERROR collecting test/unit/test_web.py ________________________________________________
ImportError while importing test module '/opt/omero-web/test/unit/test_web.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/unit/test_web.py:34: in <module>
    import omero.cli
E   ModuleNotFoundError: No module named 'omero.cli'
```

:( Looks like there are shadowing issues with `omero/*`

**Comment by joshmoore** [comment]
cf. 

```
(py36) /opt/omero-py $ pip uninstall omero-py
...
  Would not remove (might be manually added):
    /opt/anaconda/envs/py36/lib/python3.6/site-packages/omero/plugins/web.py
Proceed (y/n)? y
```

**Pull request merged**